### ISSUE TITLE
explain that wallet must be created to be able to swap

### ISSUE THREAD

**Issue by advanture-space** [asgardex/asgardex-desktop#132]
## Expected Behavior

When clicking on RUNE or the drop down menu arrow down allow to select some other coin such as ETH to swap for BTC. Even if not wallet has been created yet. I was assuming a wallet is optional.

## Current Behavior

Nothing happens when clicking on RUNE or the drop down menu arrow down.

## Steps to Reproduce

1. Download ASGARDEX-1.21.0-linux.AppImage on Debian 
2. start ASGARDEX
3. Bitcoin -> Swap
5. Click on RUNE or the drop down menu arrow down

## Possible Solution

Either make that work or show some popup or something to explain that additional steps are required. I had to create a wallet first to unblock that button.

## Your Environment - Release Version (skip if not using a release version)

- ASGARDEX desktop version: 1.21.0
- Operating system and version: Debian 12

**Comment by GrassRootsCrypto** [comment]
@advanture-space Thanks for reporting. A wallet is required to swap to sign in the input amount. 

Following the steps, doesn't it say something like "YOU NEED TO UNLOCK YOUR WALLET TO SWAP" ?
![image](https://github.com/asgardex/asgardex-desktop/assets/40199525/002af6e4-dc39-4515-8a61-3713da668ff1)


Swapping without creating or unlocking a wallet allows someone to get a quote without connecting a wallet.

**Comment by Thorian1te** [comment]
you can hit the switch asset to change the bottom to top. I agree its a little bit finicky and will get changed

**Comment by advanture-space** [comment]
I am confused. You say it will be changed but the issue is closed as wontfix.
